site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,209 @@
+---
+title: 'CVAT Overview'
+linkTitle: 'Introduction to CVAT'
+weight: 1
+description: 'CVAT: The open-source tool for image and video annotation and data labeling'
+---
+
+Machine learning systems often struggle due to
+poor-quality data. Without effective tools,
+improving a model can be tough and inefficient.
+
+[**CVAT.ai**](https://www.cvat.ai/) **is a** versatile **tool** for
+**annotating images and videos**, serving the computer
+vision community worldwide.
+
+Our goal is to help developers, businesses,
+and organizations globally by using a Data-centric AI approach.
+
+CVAT offers two versions:
+
+- [**CVAT Cloud**](https://app.cvat.ai/): Start online with CVAT,
+  **available for free**. You can also choose a [**subscription**](https://www.cvat.ai/pricing/cloud)
+  for **unlimited data**, **organizational tools**, **auto-annotations**, and [**more**](https://www.cvat.ai/post/cvat-ai-pricing-plans-choosing-the-right-plan-for-your-needs).
```

Comment:
What does "organization tools" mean? Could you please rephrase. I'm not sure that we have such term in our documentation.